packages/core/src/components/forms/formGroup.tsx

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ * Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+ * of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+ * and https://github.com/palantir/blueprint/blob/master/PATENTS
+ */
+
+import * as classNames from "classnames";
+import * as PureRender from "pure-render-decorator";
+import * as React from "react";
+import * as Classes from "../../common/classes";
+import { IIntentProps, IProps } from "../../common/props";
+
+export interface IFormGroupProps extends IIntentProps, IProps {
+    /**
+     * Whether form group should appear as non-interactive.
+     * Remember that input elements must be disabled separately
+     */
+    disabled?: boolean;
+
+    /** Optional helper text. Will be wrapped in `.pt-form-helper-text` and displayed beneath `children`. */
```

Comment:
Can we reword this? Currently, the period happens to render on its own line, which is goofy:

![image](https://user-images.githubusercontent.com/443450/31362422-472d1de8-ad0d-11e7-824a-1a45666efc37.png)

Reply:
ugh so lame! will do.